meltano.yml

```
@@ -19,11 +19,11 @@ plugins:
         keys:
         - col1
     settings:
-      - description: Array of objects with `entity`, `file`, and `keys` keys
+      - description: Array of objects with `entity`, `file`, `keys` keys, and `encoding`
```

```suggestion
      - description: Array of objects containing keys: `entity`, `file`, `keys`,  and `encoding` (Optional)
```

@seajhawk not totally related to the changes in this PR but that description needed a little cleanup. What do you think about that suggested wordin?